infrastructure/quick-deploy/aws/variables.tf

```diff
@@ -195,15 +195,17 @@ variable "keda" {
   description = "Keda configuration"
   type = object({
     namespace                       = optional(string, "default")
-    keda_image_name                 = optional(string, "ghcr.io/kedacore/keda"),
-    keda_image_tag                  = optional(string),
-    apiserver_image_name            = optional(string, "ghcr.io/kedacore/keda-metrics-apiserver"),
+    keda_image_registry             = optional(string, "")
+    keda_image_name                 = optional(string, "ghcr.io/kedacore/keda")
+    keda_image_tag                  = optional(string)
+    apiserver_image_registry        = optional(string, "")
+    apiserver_image_name            = optional(string, "ghcr.io/kedacore/keda-metrics-apiserver")
     apiserver_image_tag             = optional(string),
-    pull_secrets                    = optional(string, ""),
+    pull_secrets                    = optional(string, "")
     node_selector                   = optional(any, {})
     metrics_server_dns_policy       = optional(string, "ClusterFirst")
     metrics_server_use_host_network = optional(bool, false)
-    helm_chart_repository           = optional(string)
+    helm_chart_repository           = optional(string, "https://kedacore.github.io/charts")
```

Comment:
est ce que cette information ne pourrait pas etre extraite du versions.tfvars?

Reply:
It is indeed extracted from the versions.tfvars, but if no value is provided in the version, the default variables defined here will be used.